compiler-cli/src/config.rs

```diff
@@ -12,6 +13,35 @@ pub fn root_config() -> Result<PackageConfig, Error> {
     read(paths.root_config())
 }
 
+/// Get the config for a dependency module. Return the config for the current
+/// project if a dependency doesn't have a config file.
+pub fn module_config(mod_path: &str, project_paths: &ProjectPaths) -> Result<PackageConfig, Error> {
```

Comment:
We'll also want to ensure this is a Gleam project and not some other build tool somehow.

Reply:
Can you elaborate on what has to be done?

Reply:
I think best way will be to iterate over the packages in the manifest rather than scanning the filesystem, as then we have the information at hand and won't get confused by any other directories.